.github/workflows/prerelease.yml

```diff
@@ -33,10 +33,15 @@ jobs:
           - '3.12'
 
     steps:
-    - uses: actions/checkout@v3
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```

Comment:
Is the shallow clone necessary here, or was this a copy-paste artifact?

Reply:
0 should be unlimited, which was necessary to get the submodules from some of the errors I was resolving

Reply:
You're right, my mistake.